_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -999,9 +1000,9 @@
 	},
 /obj/machinery/power/apc{
 	cell_type = 5000;
-	dir = 8;
-	name = "west bump Important Area";
```

Comment:
I know I wiped the name of "important" APCs like this one, in another pass I'm going to make an important APC subtype if we don't already have one